std/typecons.d

```diff
@@ -3407,11 +3447,18 @@ if (Targets.length >= 1 && allSatisfy!(isMutable, Targets))
             this(       inout Source s)        inout @safe pure nothrow { _wrap_source = s; }
             this(shared inout Source s) shared inout @safe pure nothrow { _wrap_source = s; }
 
-            // BUG: making private should work with NVI.
-            protected final inout(Object) _wrap_getSource() inout @trusted
-            {
-                return dynamicCast!(inout Object)(_wrap_source);
-            }
+            static if(is(Source == class) || is(Source == interface))
+                // BUG: making private should work with NVI.
+                protected final inout(Object) _wrap_getSource() inout @trusted
```

Comment:
On a style note, these should be wrapped in curly brackets, as they span more than 1 line.